app/src/test/java/com/example/pictureupload/LoginViewModelTest.kt

```diff
@@ -61,13 +61,13 @@ class LoginViewModelTest {
 
     @Test
     fun `checking what validate returns on input incorrect`(){
```

Comment:
The name of this case could be more specific like `checking what validate returns on empty input`